### ISSUE TITLE
Create Python console example to demonstrate toplevel()

### ISSUE THREAD

**Issue by jburse** [jburse/dogelog-moon#121]
We should create a Python console example to demonstrate
the `toplevel().` The idea is that we do not really need a documentation
for the Python version of Dogelog runtime.

All we need to do is hint to the folders `cannedpy`, `runtimepy `and
the artefacts `CompressPy `and `mainpy`. What we will possibly currently
not realize for the Python version is the HTML functionality,

Although we could also do so. But the semantics of callbacks
is different between Python and JavaScript. In Python we add
a newline to the output callbacks. And possibly there willl be

more changes like adding a file handle. Nevertheless the Dogelog
runtime documentation could be made working for both JavaScript
and Python, although we might need to change our language,

when we talk about JavaScript call, replace this by host language
call. Ultimately since Python and JavaScript are in the same documentation
we could also add a Python examples. Suggestion is a console,

that also uses `bundle_clear/1` and `bundle_add/2` to produce an artefact.

**Comment by jburse** [comment]
We turned the example in not only calling toplevel(), but
also in an exercise for set_output() and set_error(). The result
is now this fancy console:

![console](https://user-images.githubusercontent.com/5411487/131257442-68d90c6b-83af-494d-938d-a8210f8a8ccf.png)

Further we could also deploy our bundler. So the top-level
consule can be made into a single Python file, relieving us
from dealing with Python modules.

**Comment by jburse** [comment]
After quite a pause we managed to update the user manual
and use a presentation that covers both the JavaScript version
and the Python version. We talk about host language which is

even open to some further other languages. We would be
now ready to document the Python console. Before we do that
we might want to do some further testing, like try the colouring

for node.js as well. We could mentiond its existence in passing
when we document the Python console. In Python os.system("")
is necessary to enable ANSI colouring.

What will be the equalivalent in node.js to explore. node.js is
problematic, the stdin is asynchronous outside of the Windows 
platform so there is more effort to have a console and there

are still open tickets. Its not clear whether we can test the
colouring on the Windows platform even, for node.js. Or 
whether we are deprieved of this possibility as well.

**Comment by jburse** [comment]
We tryed the following on the Windows platform. We didn't use `process.stdout` 
which is a WritableStream, but we used `process.stdout.fd` which allows 
synchronous execution: The finding is that this output is not buffered, so 

we do not need to place a some flush() after the writeSync():
```
function out(text) {
    fs.writeSync(process.stdout.fd, ANSI_RESET);
    fs.writeSync(process.stdout.fd, text);
    fs.writeSync(process.stdout.fd, ANSI_INPUT);
}
```
The result was:

![image](https://user-images.githubusercontent.com/5411487/132061940-f927f3b9-a4c5-4d84-9c21-7793e1d32908.png)

That the process.stdout.fd isn't buffered is rather a drawback.
Because we need 3 calls ANSI_RESET, text and ANSI_INPUT
the payload is not assembled before its send to the

terminal device. On the other hand we do not need some 
special call to configure colouring, it works by default. The
node.js page also provides some hint on how to use Unicode.

Testing Unicode for Python version and JavaScript version is
the next thing we should do. We could also mention in passing
how to use Unicode. Buffering can be spared for the write() ticket.
